swap/src/asb/config.rs

```diff
@@ -88,8 +88,8 @@ pub struct ConfigNotInitialized {}
 pub fn read_config(config_path: PathBuf) -> Result<Result<Config, ConfigNotInitialized>> {
     if config_path.exists() {
         info!(
-            "Using config file at default path: {}",
-            config_path.display()
+            default_path = %config_path.display(),
+            "Using config file at default path. ",
```

Comment:
🛠 I think this comment is actually incorrect (probably a legacy) because we are not in the context of a `default` path for the config here but this accounts for any config path passed to this function!